css/styles.css

```diff
@@ -121,5 +122,67 @@ h3 {
 }
 
 .column-half {
-  padding: 0.4rem;
+  padding: 0.2rem;
+}
+
+.delete-button {
+  text-decoration: underline;
+  color: red;
+  font-size: 0.8rem;
+  width: 10rem;
+}
+
+.modal-buttons {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 3.2rem;
+}
+
+.modal {
+  position: absolute;
+  left: 50%;
+  margin-left: -13rem;
+  top: 50%;
+  margin-top: -7rem;
+  height: 8.5rem;
+  width: 26rem;
+  background-color: white;
+  padding-left: 1rem;
+  text-align: center;
+  font-size: 1rem;
```

Comment:
There shouldn't be a need for using such large `rem` values, if you're using such large values, you're probably doing something "hacky" to get this to work. You shouldn't be needing to use width & height alongside offset values. You should be using one or the other. Revisiting `javascript-modal` would be a good exercise for this

Your modal should be split into 2. The outer div / parent div controls where it appears on the page. And then inside of that, you should have another div where all the content belongs.